frontend/src/pages/groups/GroupCreatePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { FunctionComponent } from 'react'
+import LayoutWithNav from '../../layouts/LayoutWithNav'
+import { Group, GroupInput } from '../../types/api-types'
+import { gql, useMutation } from '@apollo/client'
+import { useHistory } from 'react-router-dom'
```

Comment:
nit: here and in a couple other spots it'd be nice to keep the library imports sorted above the relative imports